clients/rust/Cargo.toml

```diff
@@ -18,5 +19,7 @@ num-derive = "^0.3"
 num-traits = "^0.2"
 serde = { version = "^1.0", features = ["derive"], optional = true }
 serde_with = { version = "^3.0", optional = true }
+solana-client = { version = "^2.1", optional = true }
+solana-sdk = { version = "^2.1", optional = true }
```

Comment:
Note that we are currently having to use these crates instead of the more granular ones to stay compatible with clients using Solana below 2.1.

We have documented a [way forward on the Codama Rust renderer here](https://github.com/codama-idl/codama/issues/277).